plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/BaseJdbcConfig.java

```diff
@@ -114,12 +146,21 @@ public BaseJdbcConfig setCacheMaximumSize(long cacheMaximumSize)
         return this;
     }
 
-    @PostConstruct
-    public void validate()
```

Comment:
Changes related to `validate` can be in a different commit or we could change them in a follow-up.